### PR TITLE
Added mvc code for order service

### PR DESCRIPTION
**Pull request by Tabish596**

### CODE REVIEW

File: server/order-service/src/main/java/com/ecomproject/order_service/controller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/order")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @PostMapping
```
Comment (sakhtar040) [inline server/order-service/src/main/java/com/ecomproject/order_service/controller/OrderController.java:16]: use this instead -> @ResponseStatus(HttpStatus.CREATED)

Comment (Tabish596) [reply server/order-service/src/main/java/com/ecomproject/order_service/controller/OrderController.java:16]: ok sir adding this

File: server/order-service/src/main/java/com/ecomproject/order_service/model/Order.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.ecomproject.order_service.model;
+
+import jakarta.persistence.*;
```
Comment (sakhtar040) [inline server/order-service/src/main/java/com/ecomproject/order_service/model/Order.java]: Optimize import

File: server/order-service/src/main/java/com/ecomproject/order_service/model/OrderLineItem.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.ecomproject.order_service.model;
+
+import jakarta.persistence.*;
```
Comment (sakhtar040) [inline server/order-service/src/main/java/com/ecomproject/order_service/model/OrderLineItem.java:3]: Optimize import

File: server/order-service/src/main/java/com/ecomproject/order_service/service/OrderService.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import com.ecomproject.order_service.model.OrderLineItem;
+import com.ecomproject.order_service.repository.OrderRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
```
Comment (sakhtar040) [inline server/order-service/src/main/java/com/ecomproject/order_service/service/OrderService.java:16]: Add logging

File: server/order-service/src/main/java/com/ecomproject/order_service/model/Order.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+@Table(name="t_orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long orderId;
+    private String orderNumber;
+
+    @OneToMany(cascade = CascadeType.ALL)
+    private List<OrderLineItem> orderLineItemsList;
```
Comment (sakhtar040) [inline server/order-service/src/main/java/com/ecomproject/order_service/model/Order.java]: correct naming

**Review by sakhtar040 — APPROVED** [review]

**Pull request merged**